bluemira/radiation_transport/flux_surfaces_maker.py

```diff
@@ -0,0 +1,316 @@
+        """
+        x_fw = np.array([fs.x_end for fs in flux_surfaces])
+        return x_fw
+
+    def _get_array_z_fw(flux_surfaces):
+        """
+        Get z_fw array of flux surface values.
+        """
+        z_fw = np.array([fs.z_end for fs in flux_surfaces])
+        return z_fw
+    
+    def _get_array_alpha(flux_surfaces):
+        """
+        Get alpha angle array of flux surface values.
+        """
+        alpha = np.array([fs.alpha for fs in flux_surfaces])
+        return alpha
+    
+    def _get_sep_out_intersection(self, outboard=True):
+        """
+        Find the middle and maximum outboard mid-plane psi norm values
+        """
+        yz_plane = self._yz_plane
+        o_point = self._o_point
+        separatrix = self.eq.get_separatrix()
+
+        if not isinstance(separatrix, Coordinates):
+            sep1_intersections = coords_plane_intersect(separatrix[0], yz_plane)
+            sep2_intersections = coords_plane_intersect(separatrix[1], yz_plane)
+            sep1_arg = np.argmin(np.abs(sep1_intersections.T[0] - o_point.x))
+            sep2_arg = np.argmin(np.abs(sep2_intersections.T[0] - o_point.x))
+            x_sep1_mp = sep1_intersections.T[0][sep1_arg]
+            x_sep2_mp = sep2_intersections.T[0][sep2_arg]
+            if outboard:
+                x_sep_mp = x_sep1_mp if x_sep1_mp > x_sep2_mp else x_sep2_mp
+            else:
+                x_sep_mp = x_sep1_mp if x_sep1_mp < x_sep2_mp else x_sep2_mp
+        else:
+            sep_intersections = coords_plane_intersect(separatrix, yz_plane)
+            sep_arg = np.argmin(np.abs(sep_intersections.T[0] - o_point.x))
+            x_sep_mp = sep_intersections.T[0][sep_arg]
+
+        out_intersections = coords_plane_intersect(self.first_wall, yz_plane)
+        if outboard:
+            x_out_mp = np.max(out_intersections.T[0])
+        else:
+            x_out_mp = np.min(out_intersections.T[0])
+
+        return x_sep_mp, x_out_mp
+
+    def _make_flux_surfaces(self, x, z):
+        """
+        Make individual PartialOpenFluxSurfaces through a point.
+        """
+        coords = find_flux_surface_through_point(
+            self.eq.x, self.eq.z, self.eq.psi(), x, z, self.eq.psi(x, z)
+        )
+        coords = Coordinates({"x": coords[0], "z": coords[1]})
+        f_s = OpenFluxSurface(coords)
+        lfs, hfs = f_s.split(self._o_point, plane=self._yz_plane)
+        return lfs, hfs
+
+    def _make_flux_surfaces_ob(self):
+        """
+        Make the flux surfaces on the outboard.
+        """
+        self.x_sep_omp, x_out_omp = self._get_sep_out_intersection(outboard=True)
+
+        self.flux_surfaces_ob_lfs = []
+        self.flux_surfaces_ob_hfs = []
+
+        start = self.x_sep_omp + self.dx_mp
+        stop = x_out_omp - EPS
+        step = ((x_out_omp - EPS - (self.x_sep_omp + self.dx_mp)) // self.dx_mp).astype(int)
+        range = np.linspace(start, stop, step)
+        for x in range:
```

Comment:
A bit simpler? The double division forces it to be an integer
```suggestion
        start = self.x_sep_omp + self.dx_mp
        for x in np.arange((x_out_omp - EPS - start) // self.dx_mp) + start:
```

Reply:
It doesn't work in that way.
Cannot I do this?
`start = self.x_sep_omp + self.dx_mp`
`stop = x_out_omp - EPS`
`for x in np.arange(start, stop, self.dx_mp):`

Reply:
Oops yeah, sure if it works but there is no need to create new variables start and stop, just do it all on one line (un;less you need start/stop): 
`for x in np.arange(self.x_sep_omp + self.dx_mp, x_out_omp - EPS, self.dx_mp):`